### PR TITLE
feat:Homework_16

### PR DESCRIPTION
**Pull request by Alexei-hub**

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import BaseComponent from "./BaseComponent";
+import LoginForm from "./LoginForm";
+
+export default class Header extends BaseComponent {
+    constructor(page) {
+        super(page, '.header');
+        this.loginForm = new LoginForm(this._page);
+        this._signInBtn = page.locator('.header_signin').last();
```
Comment (SemenSizov) [inline src/components/Header.js]: For components, we need to have a container locator as a base.
In your case - this._signInBtn = this._compomemt.locator('.header_signin').last();

File: src/components/LoginForm.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import BaseComponent from "./BaseComponent";
+import RegistartionForm from "./RegistartionForm";
+
+export default class LoginForm extends BaseComponent{
+    constructor(page) {
+        super(page, 'modal-content');
+        this.registartionForm = new RegistartionForm(this._page);
+        this._registartionBtn = page.locator('.btn-link').last();
```
Comment (SemenSizov) [inline src/components/LoginForm.js]: For components, we need to have a container locator as a base.
In your case - this._registartionBtn= this._compomemt.locator('.btn-link').last();

File: src/components/RegistartionForm.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import BaseComponent from "./BaseComponent";
+
+export default class RegistartionForm extends BaseComponent {
+    constructor(page) {
+        super(page, '.modal-dialog');
+        this.nameField = page.locator('#signupName');
+        this.lastNameField = page.locator('#signupLastName');
+        this.emailField = page.locator('#signupEmail');
+        this.passwordField = page.locator('#signupPassword');
+        this.repeatPasswordField = page.locator('#signupRepeatPassword');
```
Comment (SemenSizov) [inline src/components/RegistartionForm.js]: same comment

File: tests/registration/registartionUserWithPageObject.spec.js [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+        const mainPage = new MainPage(page);
+        await mainPage.navigateTo();
+        const loginForm = await mainPage.header.clickLoginBtn();
+        await loginForm.clickRegistrationBtn();
+    });
+
+    test('positive case. Registartion with correct data and deleting user', async ({ page }) => {
+        const registartionForm = new RegistartionForm(page);
+        await registartionForm.fillRegistartionFormFields('aqaName', 'aqaLastName', 'aqa_test@test.com', 'Aqa123456', 'Aqa123456')
+        const profilePage = registartionForm.clickRegisterBtn();
```
Comment (SemenSizov) [inline tests/registration/registartionUserWithPageObject.spec.js]: You just need to put `await` here 
const profilePage = await registartionForm.cliclRegistartionBtn()
so you don't need it later like (await profileSettingPage).removeAccountBtn.click();

Comment (Alexei-hub) [reply tests/registration/registartionUserWithPageObject.spec.js]: точно )) а я не мог понять, где ошибка. спасибо

**Pull request merged**